server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/model/BucketedSlices.java

```diff
@@ -119,5 +119,10 @@ public final boolean equals(Object o) {
         public final int hashCode() {
             return Objects.hash(startSliceInstant);
         }
+
+        @Override
+        public String toString() {
+            return "" + startSliceInstant;
```

Comment:
why use a concat to do a toString?

Reply:
This is a debug call, we can use MoreObjects here (@Arsnael  maybe?)